.travis.yml

```diff
@@ -52,6 +52,7 @@ before_install:
     conda config --set channel_priority strict
     conda config --set safety_checks disabled
     conda create --name TEST python=$PY --file requirements.txt --file requirements_dev.txt --file requirements_docs.txt --file requirements_tests.txt
```

Comment:
@weikang9009  So I should be able to completely remove both of the follow?:
 * `requirements_docs.txt` from Line 54 and;
 * `pip install 'sphinx_bootstrap_theme>=0.7.0' --force-reinstall` from Line 55